### ISSUE TITLE
 React: Super Sampler did not start correctly

### ISSUE THREAD

**Issue by wulfwuud** [VolmitSoftware/React#201]
React lastversion wont start

[14:51:26 INFO]: React: Super Sampler did not start correctly. Resetting
[14:51:26 INFO]: [React] Disabling React v6.0.5
[14:51:26 INFO]: [React] Enabling React v6.0.5
[14:51:27 INFO]: React: Language: enUS

than again same mesages React is trying to start but something going wrong

Sum errors https://hastebin.com/eqifuqevut.md

Plib 430
Via 130
Core ver Paper version git-Paper-ee2ec9fe (MC: 1.11.2)

**Comment by cyberpwnn** [comment]
```
[14:52:03 WARN]: java.lang.NullPointerException
[14:52:03 WARN]:        at react.controller.SpikeController$3.run(SpikeController.java:145)
[14:52:03 WARN]:        at react.gconcurrent.A$1.run(A.java:14)
[14:52:03 WARN]:        at react.gconcurrent.ParallelThread.execute(ParallelThread.java:68)
[14:52:03 WARN]:        at react.gconcurrent.ParallelThread.execute(ParallelThread.java:86)
[14:52:03 WARN]:        at react.gconcurrent.ParallelThread.run(ParallelThread.java:32)
```

**Comment by cyberpwnn** [comment]
Fixed in 6.2.1
